www/caddy/src/opnsense/service/templates/OPNsense/Caddy/Caddyfile

```diff
@@ -228,6 +228,8 @@
     # Default of Caddy is to wait for all connections to close before allowing reload, meaning the higher the value, the longer applies take.
     #}
     grace_period {{ generalSettings.GracePeriod }}s
+    {# Important: Prevent sudo appearing on startup for unprivileged users. https://github.com/smallstep/truststore/blob/master/truststore_freebsd.go #}
```

Comment:
IMO it's better to document this in the commit message and keep the template clean